src/PointwiseFunctions/EquationsOfState/PolytropicFluid.cpp

```diff
@@ -59,28 +59,31 @@ template <>
 template <class DataType>
```

Comment:
Please make your commit message active and split into two commits:
- `Fix polytropic equation of state's rest mass density from enthalpy`
- `Add equation of state test` then explain what it tests and why